lib/JLArrays/src/JLArrays.jl

```diff
@@ -123,6 +139,18 @@ mutable struct JLArray{T, N} <: AbstractGPUArray{T, N}
     end
 end
 
+"""
+    JLSparseVector{Tv, Ti}
+
+Sparse vector backed by `JLVector`s, similar to `SparseArrays.SparseVector`.
+
+# Fields
+
+- `iPtr::JLVector{Ti, 1}`: indices of non-zero coefficients
+- `nzVal::JLVector{Tv, 1}`: values of non-zero coefficients
+- `len::Int`: size of the vector
+- `nnz::Ti`: number of non-zero coefficients
```

Comment:
I'm personally not a fan of unconditionally documenting all fields, especially when they are implicitly set by constructors and exposed through interface methods (nnzvals, length, etc). They aren't public API, but sadly there's no way to express that.

I'd be fine doing this for some other structures, so it's not a general rule though.

Reply:
> They aren't public API, but sadly there's no way to express that.

That's why I want to be careful about this PR, because in Julia 1.10 documented = public by default. In Julia 1.11 though we could add the public keyword!